+dj/Schema.m

```diff
@@ -32,6 +32,7 @@
         %   computed: tableName with '__'
         allowedTiers = {'lookup' 'manual' 'imported' 'computed' 'job'}
         tierPrefixes = {'#', '', '_', '__', '~'}
+        tierClasses = {'dj.Lookup' 'dj.Manual' 'dj.Imported' 'dj.Computed' 'dj.Jobs'}
```

Comment:
I understand but we could go ahead and push the tier prefix information into the class while still supporting the older syntax - I don't like how the UserRelation classes are completely empty and placing tier prefix can be a useful information in the class.